packages/vaadin-lumo-styles/utilities/flexbox-grid.js

```diff
@@ -89,20 +92,20 @@ export const flexboxAndGrid = css`
   }
 
   /* === Flex grow === */
-  .flex-grow-0 {
-    flex-grow: 0;
-  }
-  .flex-grow {
+  .grow {
     flex-grow: 1;
   }
+  .grow-0 {
+    flex-grow: 0;
+  }
 
   /* === Flex shrink === */
-  .flex-shrink-0 {
-    flex-shrink: 0;
-  }
-  .flex-shrink {
```

Comment:
We should not remove old classes until the next major, as that would be a breaking change.
I would suggest keeping them in case someone relies on these in Hilla Lit / React views.

Reply:
Reverted.